src/components/Dashboard/Alerts/MissingInformation.vue

```diff
@@ -24,7 +24,11 @@
         <p class="mb-0 pt-5">
           If further action is required, please contact BC Registries staff:
         </p>
-        <ContactInfo class="pt-5" />
+        <ContactInfo
+          class="pt-5"
+          :hidePhoneTollFree="hidePhoneNumbers"
+          :hidePhoneVictoria="hidePhoneNumbers"
+        />
```

Comment:
It makes more sense to me to hide what you don't want, instead of showing what you do want.